.travis.yml

```diff
@@ -1,5 +1,27 @@
-rvm:
-  - "2.3.1"
-script: "./test_all_impls.sh"
 language: ruby
+
 sudo: false
+
+env:
+  global:
+    - SASS_LIBSASS_PATH=$TRAVIS_BUILD_DIR/../libsass
+
+matrix:
+  fast_finish: true
+  include:
+    - gemfile: Gemfile
+      env: LANGUAGE_VERSION=3.4
+    - gemfile: Gemfile.sass_3_5
+      env: LANGUAGE_VERSION=3.5
+    - gemfile: Gemfile.sass_4_0
+      env: LANGUAGE_VERSION=4.0
+
+before_install:
+  - rm Gemfile.lock
+  - git clone https://github.com/sass/libsass.git $TRAVIS_BUILD_DIR/../libsass
+  - git clone https://github.com/sass/sassc.git $TRAVIS_BUILD_DIR/../sassc
+  - make -C ../sassc
+
+script:
+  - bundle exec sass-spec.rb -V "$LANGUAGE_VERSION" --probe-todo
+  - bundle exec sass-spec.rb -V "$LANGUAGE_VERSION" --probe-todo -c "../sassc/bin/sassc" --impl 'libsass' || if [ "$LANGUAGE_VERSION" != "3.4" ]; then exit 0; fi
```

Comment:
That's what this is doing right now with this. It allows the tests to fail if it isn't running 3.4. When 3.5 becomes "stable", then I'd change this to `[ "$LANGUAGE_VERSION" = "4.0" ]` so it would only tollerate failures there.